src/PowerShellEditorServices.Protocol/Server/LanguageServer.cs

```diff
@@ -1218,6 +1221,36 @@ public static string GetDecoratedSymbolName(SymbolReference symbolReference)
             await requestContext.SendResultAsync(result);
         }
 
+        protected async Task HandleGetRunspaceRequestAsync(
+            object noParams,
+            RequestContext<GetRunspaceResponse[]> requestContext)
+        {
+            var runspaceResponses = new List<GetRunspaceResponse>();
+
+            if (this.editorSession.PowerShellContext.LocalPowerShellVersion.Version.Major >= 5)
+            {
+                var psCommand = new PSCommand();
+                psCommand.AddCommand("Get-Runspace");
+
+                var runspaces = await editorSession.PowerShellContext.ExecuteCommandAsync<PSObject>(psCommand);
```

Comment:
can you replace `var` with the type since this is a tad more complicated than `new Blah()`

Reply:
Also, instead of PSObject you should be able to do `System.Management.Automation.Runspaces.Runspace` and then you don't have to use dynamic below.

Reply:
👍